build.cake

```diff
@@ -131,25 +150,44 @@ Task("Test")
 	.IsDependentOn("Build")
 	.Does(() =>
 	{
-		NUnit3(UNIT_TEST_PATH);
+		NUnit3(BIN_DIR + UNIT_TEST_ASSEMBLY);
 	});
 
 //////////////////////////////////////////////////////////////////////
 // PACKAGE
 //////////////////////////////////////////////////////////////////////
 
-Task("Package")
+Task("PackageNuGet")
 	.IsDependentOn("Build")
```

Comment:
No... or at least not yet... This gets to the question of how we access pure repackaging. Do we want a Repackage target that doesn't depend on build? Should Package depend on Build and Repackage? etc. I elected not to pick one but to see what folks thought first.

Right now, if you want to repackage the chocolatey package, you have to enter a pretty obscure command...

```
./build.ps1 -t Package -ScriptArgs "--chocoVersion=3.5.1 --binaries=Path/to/binaries"
```

You will trigger a build but the output will not be used for chocolatey. Instead, the path you specify will provide the binaries. 

Even if we don't do it often, I think it would be nice to provide a friendlier front end for this.